src/transformers/models/musicgen_melody/modeling_musicgen_melody.py

```diff
@@ -1294,7 +1294,7 @@ def generate(
                 )
 
             # 11. run greedy search
-            outputs = self.greedy_search(
+            outputs = self._greedy_search(
```

Comment:
we're `greedy_search`/`sample` (public functions) in place of `_greedy_search`/`_sample` (private functions)